npm-init.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+
+const content = `{ 
+  "extends": "semistandard"
+}`
+fs.writeFileSync('.eslintrc.json', content);
+fs.mkdirSync('test');
+
+module.exports = {
+  name: prompt('name', basename || package.name),
+  version: '0.0.1',
+  description: prompt(s => s),
+  main: prompt('entry point', 'index.js', ep => fs.writeFileSync(ep, '')),
+  author: 'Red Hat, Inc.',
+  license: 'Apache-2.0',
+  scripts: {
+    test: 'tape test/*.js | tap-spec',
+    lint: 'eslint test/*.js index.js',
+    prepublish: 'nsp check',
+    coverage: 'istanbul cover tape test/*.js'
+  },
+  repository: {
+    type: 'git',
+    url: 'git://github.com/USER/' + basename + '.git'
+  },
+  files: [
+    'package.json',
+    'README.md',
+    'LICENSE',
+    'index.js'
+  ],
+  bugs: {url: 'https://github.com/USER/' + basename + '/issues'},
```

Comment:
Why not compute the base Github URL at the top of the script and use `process.env.USER`, then use that base here and in the couple of other places where the URL shows up (lines 24 and 33).

    const user = process.env.USER || 'USER';
    const repoUrl = `https://github.com/${user}/${basename}`;

Then,

    bugs: {url: `${repoUrl}/issues`},